create_db.py

```diff
@@ -0,0 +1,6 @@
+from model import Base
+from db_conection import engine
+
+if __name__ == "__main__":
+    Base.metadata.drop_all(engine)
```

Comment:
No, this is for creating the database. This is intended to run only once. This will simply create the tables in the ORM. I guess I can ask the user if he wants to continue if the table already exists.

Reply:
Okay, got it. No changes required.